src/containers/CurrentImageContainer.js

```diff
@@ -0,0 +1,23 @@
+import { connect } from 'react-redux';
+import { CurrentImage } from '../views/CurrentImageView';
+
+function mapStateToProps(state) {
+  return {
+    currentImage: state.currentImage,
+  };
+}
+
+function mapDispatchToProps(dispatch, ownProps) {
+  let { selectImage } = ownProps;
+  if (!selectImage) {
+    selectImage = () => null;
+  }
+  return {
+    selectImage: selectImage,
+  };
+}
```

Comment:
in this `mapDispatchToProps`, we're not using `dispatch` at all, so we could move this logic into the `CurrentImage` component and set a defaultProp for `selectImage` if it isn't defined